src/Components/LoginPage/LoginPageComponents/SignButton.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const SignButton = (props) => {
+    if (props.text) return (
```

Comment:
please use destructuring` ({text}) => `

Reply:
Oops... this file don't using in current version and i delete it